syntax = "proto2";
package openscenario;

// Enum for Rule
enum Rule {
    EQUAL_TO = 0;
    GREATER_THAN = 1;
    LESS_THAN = 2;
    GREATER_OR_EQUAL = 3;
    LESS_OR_EQUAL = 4;
    NOT_EQUAL_TO = 5;
}
